site/src/components/pages/user/block-list-item.tsx

```diff
@@ -18,10 +18,10 @@ export const BlockListItem = ({ block }: BlockListItemProps) => {
       title={block.displayName!}
       description={block.description}
       extraContent={
-        <Box display="flex" gap={1.5}>
-          <Link href={`/@${block.author}`}>@{block.author}</Link>
+        <Box display="flex" gap={1.5} component="span">
```

Comment:
Each list item on the `/@username/blocks` page is a link (`<a>`):

<img width="1485" alt="Screenshot 2022-09-27 at 19 31 57" src="https://user-images.githubusercontent.com/608862/192607886-0d23ee2b-4381-4e07-b4de-63023bf768a0.png">

This means that we cannot use `<div>`, `<p>` or `<a>` components inside list items. I switched to `span` in all cases to fix hydration issues.

Reply:
I'm not sure if we should get rid of the link, or render it without an `<a>` tag to keep the same design. I feel like this is a design choice.